### ISSUE TITLE
Use Pekko instead of Akka

### ISSUE THREAD

**Issue by adpi2** [lunatech-labs/lunatech-scala-2-to-scala3-course#105]
Because [Akka is not open source anymore](https://www.lightbend.com/akka/license-faq) maybe the course should use [Pekko](https://github.com/apache/incubator-pekko), which is the open source fork of Akka, but Pekko is not yet released.

**Comment by eloots** [comment]
Good point, but maybe we should wait for a first release.

**Comment by eloots** [comment]
@adpi2 

Update: "porting" the akka code to pekko is a misnomer: it's actually trivially simple!

So, we just need to decide if we want to do it or not. Or we we turn it into an exercise (although it's not Scala 3 specific).

**Comment by adpi2** [comment]
> So, we just need to decide if we want to do it or not. Or we we turn it into an exercise (although it's not Scala 3 specific).

I am in favor of porting to Pekko. It is better if we use the latest version of a library, but the latest version of Akka is under business source license.

**Comment by eloots** [comment]
Porting to Pekko is _very_ easy. We could even make it an exercise. WDYT?

**Comment by adpi2** [comment]
> We could even make it an exercise. WDYT?

I think we should focus on the migration to Scala 3, and exclude the migration to Pekko from the exercises. Otherwise it could be misinterpreted as: "We need to migrate to Pekko because of Scala 3".

We can explain the difference between Pekko and Akka, and the reason why we chose Pekko. We can explain that it is really easy to migrate from Akka to Pekko, but more as a side note than an exercise.

**Comment by eloots** [comment]
Makes sense. I'll migrate from Akka to Pekka then. Also need to change references to Akka to Pekko in the slide deck.
